qa/suites/fs/functional/tasks/damage.yaml

```diff
@@ -19,6 +19,7 @@ overrides:
       - MDS_READ_ONLY
       - force file system read-only
       - with standby daemon mds
+      - MDS abort because newly corrupt dentry
```

Comment:
What if there is an actual corruption with one of the other damage tests? Would it get treated as an expected scenario?

Reply:
It will be ignored for this test, yes.

Reply:
Hm.. Its probably fine since we hope that the corruption would get reproduced in one of the postgresql tests.

Reply:
> Hm.. Its probably fine since we hope that the corruption would get reproduced in one of the postgresql tests.

Is it possible that a corruption that we expected in postgresql tests is produced here? We won't be able to figure that out now right if that is a case?

Reply:
> > Hm.. Its probably fine since we hope that the corruption would get reproduced in one of the postgresql tests.
> 
> Is it possible that a corruption that we expected in postgresql tests is produced here? We won't be able to figure that out now right if that is a case?

No.